### ISSUE TITLE
[Legitimate Site Blocked] luckydoods.com

### ISSUE THREAD

**Issue by 100FIRELTD** [MetaMask/eth-phishing-detect#11570]
https://luckydoods.com/

We are a legitimate NFT collection. We have not even started our minting. Please unblock us ASAP!

**Comment by 100FIRELTD** [comment]
We are not listed on any scam sites. I really don't understand why we are flagged. Please respond ASAP

**Comment by AlexHerman1** [comment]
Please post any relevant links to establish the identity of your brand and site. Thank you

**Comment by 100FIRELTD** [comment]
You can find it on our social media:

https://discord.com/invite/kWRDbsrEkr
https://twitter.com/luckydoods

On third party site below you can just search Lucky Doods:

https://rarity.tools/upcoming

**Comment by 100FIRELTD** [comment]
Hi this issue is urgent. There is no basis to flag our website. Your own testing tool shows there is no phishing issue. We are now, consulting with our legal department to seek compensation and punitive damages as we now need to postpone our product launch. We expect an immediate resolution to this matter.

**Comment by shoorsamridhi** [comment]
> Please post any relevant links to establish the identity of your brand and site. Thank you

Dear ALEX!

I understand that you are trying to make internet/blockchain a secure place for everyone but flagging accounts like this without any responsibility of the damages that another legit entity/person may incur isn't the right way to go about things. This act/initiative of yours is liable to damages & considered unlawful as a form of digital harassment & intentional obstruction to other person's entity business/work. 

Kindly, make amends at your earliest. This has been a great inconvenience!!

**Comment by AlexHerman1** [comment]
Hi, going to get this addressed today

**Comment by AlexHerman1** [comment]
fixed by #11580

**Comment by 100FIRELTD** [comment]
Thank you @shoorsamridhi.

@AlexHerman1 we have rescheduled our sale date and still have not yet seen the change take effect on Metamask. 

Please understand that the postponement of our launch to an undetermined date will cause us irreparable reputational damage. Please sort this out IMMEDIATELY.

**Comment by AlexHerman1** [comment]
Hi there,

the change takes about an hour to take effect. It is not being blocked on my end anymore. Please be patient, thank you.

**Comment by johnnyisadegen** [comment]
@AlexHerman1 the fact that you have seriously defamed this company and you have yet to apologise in any way speaks volumes of your character. What you did here, holding their business hostage like this for no apparent reason is not only pursuable for financial damages, but illegal and criminally punishable in many jurisdictions. 

 As far as I can tell there has been no complaint against this company and their products. Their Web3 functionality has not even been made publicly available on their website yet. There is absolutely no reason to suspect them of any wrong doing.

A quick review of previous and even open tickets involving similar issues with other parties, shows a seriously deranged dev on a power trip. In each instance you seem to conjure up arbitrary requirements for projects to prove that they are legitimate, while there are no obvious reasons to think that they are involved in anything shady. Some tickets seem to remain unresolved for days or even indefinitely.

I look forward to reading the entire archive of the git issues on this repo, when they are admitted into evidence in the inevitable class action suit against you, ConsenSys, and Metamask that will result from your utter stupidity.
